.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,13 +13,14 @@
 #### Type of Improvement
 
 <!-- Choose one or more from the following:
-- bug fix
-- performance improvement
-- documentation
-- code refactoring
-- code cleanup
-- backend code generation
-- new feature/API
+- Feature (or new API)
+- Performance improvement
+- API modification
+- API deprecation
+- Bugfix
+- Documentation or website-related
+- Dependency update
+- Internal or build-related (includes code refactoring/cleanup)
 -->
 
 #### API Impact
```

Comment:
should these be labels too? Or say something like "if you picked API Modification above, explain more here"?

Reply:
same with below "Backend Code Generation Impact" -- should that be a label?

Reply:
And can we get rid of "desired merge strategy"...? 

Mostly trying to slash the words people write that are redundant with labels they will apply.

Mayve this is getting too demanding for this PR :)

Reply:
> And can we get rid of "desired merge strategy"...?
> 
> Mostly trying to slash the words people write that are redundant with labels they will apply.
> 
> Mayve this is getting too demanding for this PR :)

I'm all for some streamlining of the template, but I would rather save that for a separate PR

Reply:
Actually I am going to tweak it a bit because things like API and Backend Code Generation Impact should be included in release notes and thus shouldn't be separate sections.